frontend/src/containers/LibraryPage/LibraryPage.js

```diff
@@ -107,67 +137,106 @@ class LibraryPage extends React.Component {
       }
     };
 
+    const renderDownloadSnackbars = () => {
+      if (showDownloadError) {
+        return (
+          <CustomSnackbar
+            message="Ooops.. Something wrong happened. Please try again later"
+            onClose={() => this.setState({ showDownloadError: false })}
+            severity="error"
+            topCenter
+          />
+        );
+      }
+      if (showDownloadInProgress) {
+        return (
+          <CustomSnackbar
+            message="Please wait. We are crunching the videos for you"
+            severity="info"
+            topCenter
+          />
+        );
+      }
+      if (showDownloadSuccess) {
+        return (
+          <CustomSnackbar
+            message="Videos downloaded successfully"
```

Comment:
Same here. Also, since this snackbar is shown, when zipping finishes, and browser starts downloading videos, then maybe the text should be: "Video has started downloading"